universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -0,0 +1,100 @@
+package views.components;
+
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.input;
+import static j2html.TagCreator.label;
+import static j2html.TagCreator.textarea;
+
+import j2html.attributes.Attr;
+import j2html.tags.ContainerTag;
+import j2html.tags.Tag;
+import views.BaseStyles;
+import views.Styles;
+
+public class FieldWithLabel {
+  public static String[] CORE_FIELD_CLASSES = {
+    BaseStyles.FIELD_BACKGROUND_COLOR,
+    BaseStyles.FIELD_BORDER_COLOR,
+    Styles.BLOCK,
+    Styles.BORDER,
+    Styles.P_2,
+    Styles.W_FULL
+  };
+  public static String[] CORE_LABEL_CLASSES = {
```

Comment:
Suggestion (blocking): make `CORE_FIELD_CLASSES` and `CORE_LABEL_CLASSES` `private static final`.

Reply:
Done.